src/types/venues.ts

```diff
@@ -171,6 +171,7 @@ export interface BaseVenue {
   showRadio?: boolean;
   showBadges?: boolean;
   showZendesk?: boolean;
+  scheduleStartDate?: firebase.firestore.Timestamp;
```

Comment:
Pending answer as to why we are adding a new field here at all (rather than using existing), as per https://github.com/sparkletown/sparkle/pull/1419#discussion_r635710738 + https://github.com/sparkletown/sparkle/pull/1419#discussion_r635709031

Reply:
I've now modified this PR to use the `start_utc_seconds` value

Reply:
Cool, sounds good.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1419/commits/b78373170c83bbc19324c8694b04ea35b9ab06ef